### ISSUE TITLE
Can it recognize some text or number in a specific position in a picture?

### ISSUE THREAD

**Issue by NataliaAssange** [JaidedAI/EasyOCR#813]
### Discussed in https://github.com/JaidedAI/EasyOCR/discussions/812

<div type='discussions-op-text'>

<sup>Originally posted by **NataliaAssange** August  5, 2022</sup>
As you know, it can output the text recognized and the position of the text. Can it be reversed, like, I input the position and you output the text?
Thank you!</div>

**Comment by rkcosmos** [comment]
yes, use method `recognize`. See https://www.jaided.ai/easyocr/documentation/. The inputs are image and bounding boxes, output is text.
